pkg/importer/dataStream.go

```diff
@@ -913,7 +929,7 @@ func (d *DataStream) copy(dest string) error {
 		// Replace /data/target name with scratch path/target name
 		tmpDest := filepath.Join(d.ScratchDataDir, filepath.Base(dest))
 
-		err := StreamDataToFile(d.topReader(), tmpDest)
+		err := StreamDataToFile(d.topReader(), tmpDest, d.VolumeMode, true)
```

Comment:
Since we know at this point, tmpDest is the scratch dir (always, as this is the scratch space needed path), you are passing the VolumeMode of the tmpDest to StreamDataToFile, which is always file system. So you don't need the extra boolean flag.

In this case you always pass file system volume mode, regardless of the target being block or not (the convertQcow2ToRaw will copy to whatever target we have)